explore-firewalls.md

```diff
@@ -34,9 +34,9 @@ Scroll to the right to view the rest of the table!
 |**Multi-VLAN Support**|No|Yes|Yes|Yes|No|No|No|
 |**NAT Support**|No|Yes|Yes|Yes|Yes|No|No|
 |**SSL/IPsec VPN Termination**|No|Yes|Yes|Yes|Yes|No|No|
-|**Open VPN Termination**|No|Yes|Yes|No|No|No|No|
+|**Open VPN Termination**|No|No|Yes|No|No|No|No|
 |**HA Option**|N/A|Yes|Yes|Yes|Yes|No|Yes|
 |**Manage from API & Portal**|Yes|Appliance GUI|Appliance GUI|Appliance GUI|Appliance GUI|Yes|Yes|
 |**10Gbps Support**|N/A|Yes|Yes|Yes|No|No|No|
-|**NGFW Add-ons (IPS, AV, WF)**|No|No|No|Yes|Yes|No|No|
+|**NGFW Add-ons (IPS, AV, WF)**|No|Yes(2019)|No|Yes|Yes|No|No|
```

Comment:
Should we say 2019 here or is it implied that the capability exists today? or maybe we should say **New!!**

Reply:
@brandon-wright ^

Reply:
@ibm-cos-help -- We're just trying to highlight that the product supports these features and we're looking to provide access to them via add-on in 2019. Was trying to keep it simple for the matrix. Any thoughts on how to better convey this?

Reply:
If the product supports it now, then a "Yes" should suffice, without a year. If there are caveats to the support, then let's add a few stars and a footnote. @jbmitch should be able to help with that